docs/src/documentation/03-components/09-text/heading/03-accessibility.mdx

```diff
@@ -4,25 +4,25 @@ redirect_from:
   - /components/heading/accessibility/
 ---
 
-# Accessibility
-
-## Heading
+## Accessibility
 
 The Heading component has been designed with accessibility in mind.
 
 The component offers flexibility in terms of the HTML element used for the root node, the `role` attribute, and the `level` attribute.
 These properties allow for the creation of semantic and accessible headings.
 
-| Name  | Type                                                    | Description                                                                                                                 |
-| :---- | :------------------------------------------------------ | :-------------------------------------------------------------------------------------------------------------------------- |
-| as    | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6" \| "div"` | Defines the HTML element to be rendered.                                                                                    |
-| role  | `"heading"`                                             | Can only be used if `as="div"`. If defined, sets the role of the element to be "heading".                                   |
-| level | `number`                                                | Can only be used if `as="div"` and "`role="heading`". Defines the `aria-level` of the rendered `div` with the heading role. |
+| Name  | Type                                                    | Description                                                                                                                |
+| :---- | :------------------------------------------------------ | :------------------------------------------------------------------------------------------------------------------------- |
+| as    | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6" \| "div"` | Defines the HTML element to be rendered as root node.                                                                      |
```

Comment:
I feel having `as root node` is more confusing that nothing. A Heading is not supposed to be a nesting of things, so I think it's pretty straightforward that it refers to the main element node

Reply:
I guess it should be "Defines the HTML element to be rendered" also here then? https://orbit.kiwi/components/text/heading/react/#props

Reply:
IMO, absolutely